mentoring/public/js/mentoring_assessment_view.js

```diff
@@ -41,10 +46,14 @@ function MentoringAssessmentView(runtime, element, mentoring) {
 
         mentoring.renderAttempts();
 
-        if (data.assessment_message && data.num_attempts < data.max_attempts) {
+        if (data.assessment_message && ! no_more_attempts()) {
             mentoring.setContent(messagesDOM, data.assessment_message);
             messagesDOM.show();
         }
+        $('a.question-link', element).each(function (){
```

Comment:
I don't think `.each(...)` is needed here, you could just do `$('a.question-link', element).click(jumpToName);`.